src/Microsoft.VisualStudio.AppDesigner/PropPages/PropPageUserControlBase.vb

```diff
@@ -3337,7 +3337,14 @@ NextControl:
         ''' <returns></returns>
         ''' <remarks></remarks>
         Protected Function GetProjectPath() As String
-            Return CStr(GetCommonPropertyValueNative("FullPath")) 'CONSIDER: This won't work for all project types, e.g. ASP.NET when project path is a URL
+            Dim PropertyName = "FullPath"
+            Dim propertyDescriptor = GetCommonPropertyDescriptor(PropertyName) 'CONSIDER: This won't work for all project types, e.g. ASP.NET when project path is a URL
+
```

Comment:
Before we make changes like this I'd like a understanding of what the behavior change this will cause for existing projects.